docs/releases/v2019.1.rst

```diff
@@ -0,0 +1,235 @@
+Gluon 2019.1
+############
+
+Important notes
+***************
+
+Gluon v2019.1.x will be the last series of releases to support both batman-adv-legacy (v14 compat)
+and IBSS (Ad-hoc) mesh links. Migration features have been developed and should be used during this
+release cycle to migrate to batman-adv v15 compat and/or 802.11s mesh links. These migration features
+are the :doc:`scheduled domain switching <../package/gluon-scheduled-domain-switch>` (since v2018.2.1)
+and :ref:`batman-adv module coexistence <release-v2019.1-batman-adv-coexistence>` (since v2019.1, see below).
+Failure to migrate will prevent an upgrade to Gluon releases v2019.2 and later.
+
```

Comment:
please help users understand why support is dropped for ibss (new device class, support for OWA, support for multiple radios) and for batman (unmaintained and as result won't properly build on kernels younger than 2017)

Reply:
Let's wait for the discussion in #1747 to settle.

Reply:
We discussed to keep the release notes short and sweet and leave the reasoning out of the release notes, as ultimately it does not matter much to the average joe.